src/modules/featureproperties/featureproperties-directive.js

```diff
@@ -98,12 +91,12 @@ angular.module('anol.featureproperties')
                                         if (translatedValue === translateValue) {
                                             translatedValue = value;
                                         }
-                                        properties[key] = {
-                                            key: translatedKey,
-                                            value: translatedValue
-                                        };
+                                        key = translatedKey;
+                                        value = translatedValue;
                                     }
                                 );
+                                return { key: key.includes('.') ? key.split('.').pop() : key,
```

Comment:
I'm not sure if this works as expected. Since `$translate` seems to be asynchronous, you always return a promise here, instead of the resolved values.